### ISSUE TITLE
[Feature Request] Disable status-bar using keybinding

### ISSUE THREAD

**Issue by sgon00** [zellij-org/zellij#2143]
I found this https://github.com/zellij-org/zellij/discussions/955, but it's in discussion so I create an feature request here to make sure the issue will be considered.

I am new to zellij and just use it for two days. So I need the status-bar to guide me because I can not remember all the keybindings. But I don't want to see them when I don't need to. I would like to hide them on the fly in most time and only show the status-bar when I need some guides.

Thanks a lot.

**Comment by nnzv** [comment]
I agree that having a status-bar in zellij that can be hidden when not needed, but quickly accessed when guidance on keybindings is required, would be a helpful addition. It's worth mentioning that tmux already has a similar feature, which could provide a useful reference for implementing this in zellij:

```tmux
set -g status off
```

**Comment by ancion** [comment]
I really need this feature too!!

**Comment by hawkinchina** [comment]
it is a good feature request.
